### ISSUE TITLE
bug on AnonymousSocks5Server.checkIoException(final Exception e)

### ISSUE THREAD

**Issue by dongyuanlongwang** [bbottema/simple-java-mail#56]
when I use in window10 , jdk is jdk1.8.0_25, this is work fine.
but when I use in linux centos 6.5,jdk is 1.8.0_91,It throw a exception 

```
Exception in thread "Thread-6" org.simplejavamail.mailer.internal.socks.common.SocksException: server crashed...
    at org.simplejavamail.mailer.internal.socks.socks5server.AnonymousSocks5Server.checkIoException(AnonymousSocks5Server.java:91)
    at org.simplejavamail.mailer.internal.socks.socks5server.AnonymousSocks5Server.run(AnonymousSocks5Server.java:76)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketException: Socket closed
    at java.net.PlainSocketImpl.socketAccept(Native Method)
    at java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:409)
    at java.net.ServerSocket.implAccept(ServerSocket.java:545)
    at java.net.ServerSocket.accept(ServerSocket.java:513)
    at org.simplejavamail.mailer.internal.socks.socks5server.AnonymousSocks5Server.run(AnonymousSocks5Server.java:72)
    ... 1 more

```

I debug this ,find in windows ,**e.getMessage()** is **"socket closed"** ,but in linux , is **"Socket closed"**

**Comment by bbottema** [comment]
Thanks, I will look into it.

**Comment by bbottema** [comment]
Released in v4.1.2
